src/com/facebook/buck/android/AndroidBinaryDescription.java

```diff
@@ -239,9 +239,15 @@ public BuildRule createBuildRule(
       DexSplitMode dexSplitMode = createDexSplitMode(args, exopackageModes);
 
       PackageType packageType = getPackageType(args);
+      boolean shouldProguard =
```

Comment:
Is it possible to unify this with AndroidBinaryBuildable.shouldProguard()

Reply:
Just noticed this. It might be possible, but the logic is fairly trivial and not sure if its worth DRY'ing up